css/main.css

```diff
@@ -0,0 +1,33 @@
+nav {
+  display: flex;
+}
+
+ul {
+  width: 100%;
+  list-style-type: none;
```

Comment:
very small note that I learned from doing the calculator project. Google's style guide has [CSS declarations](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order) in alphabetical order.